apps/platform-app/src/app/[locale]/(auth)/auth.helpers.ts

```diff
@@ -13,17 +14,32 @@ export const authorizeUser = async () => {
 
 export const getAuthorizedUser = async () => {
   const session = await auth()
+
   if (!session || !session.user) {
-    return { user: null, userIsHunter: false, userIsModerator: false }
+    return {
+      user: null,
+      userIsHunter: false,
+      userIsSpecialist: false,
+      userIsModerator: false,
+      provider: null,
+    }
   }
 
   const userIsHunter = session?.user?.roles.includes(Role.HUNTER)
+  const userIsSpecialist = session?.user?.roles.includes(Role.SPECIALIST)
   const userIsModerator = session?.user?.roles.includes(Role.MODERATOR)
   const userHasProfile = !!session?.user?.profileId
 
-  return { user: session.user, userIsHunter, userIsModerator, userHasProfile }
+  return {
+    user: session.user,
+    userIsHunter,
+    userIsSpecialist,
```

Comment:
**suggestion**: Consider using a more descriptive name for `userHasRole` to clarify its purpose.